table/block_based/filter_policy.cc

```diff
@@ -1790,8 +1785,8 @@ Status FilterPolicy::CreateFromString(
     } else {
       bloom_before_level = ParseInt(trim(value.substr(pos + 1)));
     }
-    double bloom_equivalent_bits_per_key =
-        ParseDouble(trim(value.substr(kRibbonName.size(), pos)));
+    double bloom_equivalent_bits_per_key = ParseDouble(
+        trim(value.substr(kRibbonName.size(), pos - kRibbonName.size())));
```

Comment:
Are we fixing the previous `value.substr(kRibbonName.size(), pos)`, which should have been `value.substr(kRibbonName.size(), pos - kRibbonName.size())`but was harmless due to trim()?